src/App.js

```diff
@@ -11,18 +12,19 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      selectedMovie:'',
-      error: ''
+      selectedMovie: null,
+      error: null
```

Comment:
I like the use of `null` over an empty string here! Especially for `selectedMovie`, since it is not a string.